weldx/schemas/weldx.bam.de/weldx/core/generic_series_free_dimension-0.1.0.yaml

```diff
@@ -0,0 +1,24 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://weldx.bam.de/weldx/schemas/core/generic_series_free_dimension-0.1.0"
+
+title: |
+  Schema that describes a free dimension of an expression based generic series.
+description: |
+  Description of a dimension.
+
+type: object
+properties:
+  symbol:
+    description: |
+      The symbol of the dimension that is used in the expression. If the dimension name and symbol are identical, this
+      property can be omitted
+    type: string
+  units:
+    description: |
+      A unit that represents the dimensions unit dimensionality.
+    tag: "asdf://weldx.bam.de/weldx/tags/units/units-0.1.*"
```

Comment:
Do I understand it correctly that we wouldn't need this schema/tag if we omit the mapping between name and symbol here?
And we could simply do a name -> units mapping with `wx_property_tag: "asdf://weldx.bam.de/weldx/tags/units/units-0.1.*"` ?

In that case maybe we implement this functionality using the `_alias_names` ?